package.json

```diff
@@ -72,6 +72,12 @@
     ]
   },
   "dependencies": {
+    "@ceramicnetwork/core": "^1.2.0",
+    "@ceramicnetwork/http-client": "^1.0.7",
+    "@ceramicnetwork/stream-tile": "^1.0.7",
+    "dids": "^2.4.0",
+    "key-did-provider-ed25519": "^1.1.0",
+    "key-did-resolver": "^1.2.1",
```

Comment:
looks like some dependencies are not placed properly.
I believe these should be in the `@veramo/ceramic` package.

Reply:
These dependencies are needed for integration tests. They don't belong in `@veramo/ceramic` package